Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -174,11 +174,85 @@ private extension Puck2DConfiguration {
         topImage ?? UIImage(named: "location-dot-inner", in: .mapboxMaps, compatibleWith: nil)!
     }
 
-    var resolvedBearingImage: UIImage {
-        bearingImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
+    var resolvedShadowImage: UIImage {
+        shadowImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
     }
 
     var resolvedScale: Value<Double> {
         scale ?? .constant(1.0)
     }
 }
+
+public extension Puck2DConfiguration {
+    // Create a Puck2DConfiguration instance with or without an arrow bearing image. Default without the arrow bearing image.
+    static func createDefault2DPuck(withBearing: Bool = false) -> Puck2DConfiguration {
```

Comment:
```suggestion
    static func makeDefault(showBearing: Bool = false) -> Puck2DConfiguration {
```
From the [Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/):

<img width="752" alt="Screen Shot 2022-01-14 at 8 25 59 AM" src="https://user-images.githubusercontent.com/205658/149531105-e6796bc4-079a-4b4b-822d-6b6edfbb7514.png">